sql/plan/insert.go

```diff
@@ -78,6 +99,9 @@ func (p *InsertInto) Execute(ctx *sql.Context) (int, error) {
 		}
 
 		if !found {
+			if !f.Nullable {
```

Comment:
This isn't right -- some non-nullable columns will have a default value that will be set if none is provided in an insert. Integrators will need to catch this on their own, I think. I don't think you can fix this completely on this pass.